modules/op-about-workspace.adoc

```diff
@@ -65,8 +70,8 @@ spec:
 <2> Definition of Tasks used in the Pipeline. This snippet defines two Tasks, `build-image` and `apply-manifests`, which share a common Workspace.
 <3> List of Workspaces used in the `build-image` Task. A Task definition can include as many Workspaces as it requires. However, it is recommended that a Task uses at most one writable Workspace.
 <4> Name that uniquely identifies the Workspace used in the Task. This Task uses one Workspace named `source`.
-<5> Name of the Pipeline Workspace used by the Task. Note that the Workspace `source` in turn uses Pipeline Workspace `shared-workspace`.
-<6> List of Workspaces used in `apply-manifests` Task. Note that this Task shares the `source` Workspace with `build-image` Task.
+<5> Name of the Pipeline Workspace used by the Task. Note that the Workspace `source` in turn uses the Pipeline Workspace - `shared-workspace`.
```

Comment:
I would avoid the `-` in this sentence and say something like, "...the Pipeline Workspace named `shared-workspace`.